### PR TITLE
updated view profiles class - raised bug

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@peterp please review the app.
Thanks

File: molo/profiles/templates/profiles/viewprofile.html [hunk 1]
```diff
@@ -6,10 +6,12 @@
 <h1 class="heading heading--profiles">{% trans "Hello " %}{{ user.username }}</h1>
 <div class="profiles">
  
-    <p class="profile__alias">{% trans "As you type your name comment will appear as:" %}
-    {{ user.profile.alias | default:" Anonymous " }}</p>
+    <p>{% trans "As you type your name comment will appear as:" %}
+    <span class="profile__alias--label">{{ user.profile.alias | default:" Anonymous " }}</span>
```
Comment (peterp) [inline molo/profiles/templates/profiles/viewprofile.html]: Shouldn't the line above this be label?

File: molo/profiles/templates/profiles/viewprofile.html [hunk 2]
```diff
@@ -6,25 +6,25 @@
...
 <div class="profiles">
  
-    <p>{% trans "As you type your name comment will appear as:" %}</p>
-    <label>{{ user.profile.alias | default:" Anonymous " }}</label> 
+    <p class="profile__alias">{% trans "As you type your name comment will appear as:" %}
+    {{ user.profile.alias | default:" Anonymous " }}</p>
 
-    <label>{% trans "Date of Birth: " %}</label>
-    <p class="profile__alias">{% if user.profile.date_of_birth %}
+    <p class="profile__alias">{% trans "Date of Birth: " %}
```
Comment (peterp) [inline molo/profiles/templates/profiles/viewprofile.html]: This should probably be called `profile__dob`

File: molo/profiles/templates/profiles/viewprofile.html [hunk 3]
```diff
@@ -6,25 +6,25 @@
...
         {{ user.profile.date_of_birth | date:"M d, Y" }}
       {% else %}
         {% trans " Not set." %} 
       {% endif %}
     </p>
 
     {% if settings.profiles.UserProfilesSettings.show_mobile_number_field %}
-      <label>{% trans "Mobile Number: " %}</label>
-      <p class="profile__alias">{{ user.profile.mobile_number | default:"Not set" }}</p>
+      <p class="profile__alias">{% trans "Mobile Number: " %}
```
Comment (peterp) [inline molo/profiles/templates/profiles/viewprofile.html]: `profile__mobile-number`

File: molo/profiles/templates/profiles/viewprofile.html [hunk 4]
```diff
@@ -6,25 +6,25 @@
...
+      {{ user.profile.mobile_number | default:"Not set" }}</p>
     {% endif %}
 
   {% if settings.profiles.UserProfilesSettings.show_email_field %}
-    <label>{% trans "Email Address:" %}</label>
-    <p class="profile__alias">{{ user.email | default:"Not set" }}</p>
+    <p class="profile__alias">{% trans "Email Address:" %}
```
Comment (peterp) [inline molo/profiles/templates/profiles/viewprofile.html]: `profile__email`

**Comment by Mitso** [comment]
@peterp I've updated the HTML page above. Please let me know if happy.

**Pull request merged**